tests/test_cfht.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of obs_metadata.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+
+from helper import readTestFile, UsefulAsserts
+from lsst.obs.metadata import ObservationInfo
+
+
+class MegaPrimeTestCase(unittest.TestCase, UsefulAsserts):
+
+    def testMegaPrimeTranslator(self):
+        test_data = (("fitsheader-megaprime.yaml",
+                      dict(telescope="CFHT 3.6m", instrument="MegaPrime",
+                           abstract_filter="i")),
+                     )
+        for file, expected in test_data:
+            self.assertObservationInfo(file, **expected)
+
+    def testMegaPrimeStripping(self):
+        header = readTestFile("fitsheader-megaprime.yaml")
+        v1 = ObservationInfo(header)
+
+        # Check that headers have been removed
+        newHdr = v1.strippedHeader()
+        self.assertNotIn("INSTRUME", newHdr)
+        self.assertNotIn("TELESCOP", newHdr)
+        self.assertIn("CCD", newHdr)
```

Comment:
Are these really the only values that get stripped?

Reply:
No. I wanted to test that stripping was working at all.  It's only recently that the translators have stabilized so I'm not sure whether testing that everything is identical to what I am currently getting will tell me much. It won't tell me I'm doing the translations correctly as such...  For example, tests did not show that obs_decam is putting the wrong longitude in VisitInfo at the moment, or that the air pressure for hsc is the wrong units.